pdf_craft/transform.py

```
@@ -1,13 +1,14 @@
 from os import PathLike
 from pathlib import Path
-from typing import Callable, Container, Literal
+from typing import Callable, Container, Literal, NoReturn
 
 from epub_generator import BookMeta, LaTeXRender, TableRender
 
 from .common import EnsureFolder, remove_surrogates
 from .error import (
     IgnoreOCRErrorsChecker,
     IgnorePDFErrorsChecker,
+    InterruptedError,
```

Ruff (0.16.1)

[error] Import `InterruptedError` is shadowing a Python builtin

(A004)